drivers/builtin/src/pk_wrap.c

```diff
@@ -120,20 +120,6 @@ int  mbedtls_pk_psa_rsa_sign_ext(psa_algorithm_t alg,
     return PSA_PK_TO_MBEDTLS_ERR(status);
 }
 
-static int rsa_sign_wrap(mbedtls_pk_context *pk, mbedtls_md_type_t md_alg,
```

Comment:
This comment refers to `mbedtls_pk_psa_rsa_sign_ext` above. Why didn't you embed it into `mbedtls_pk_sign_ext()` in the same way as you've done with other signature functions?
Is there perhaps another issue for the `sign_ext` function? If so then sorry, but I didn't find it.